src/js/views/globals/app-nav/app-nav_views.js

```diff
@@ -12,6 +12,14 @@ const i18n = intl.globals.appNav.appNavViews;
 
 const MainNavDroplist = Droplist.extend({
   popWidth: '248px',
+  position() {
+    const { outerHeight } = this.getView().getBounds();
+
+    return {
+      top: outerHeight,
+      left: 16,
+    };
+  },
```

Comment:
This adds `left: 16px` to the  main nav droplist to match the design docs.

Old:

![Screenshot 2023-01-17 at 2 10 14 PM](https://user-images.githubusercontent.com/35355575/213001736-991f8744-a2b3-4942-bdba-dfc28a6ab711.png)

New: 

![Screenshot 2023-01-17 at 2 10 24 PM](https://user-images.githubusercontent.com/35355575/213001748-001dd85b-b99b-497a-a006-bcd40858db1d.png)